src/main/webapp/WEB-INF/tags/shared/commentsPanel.tag

```diff
@@ -21,65 +21,89 @@
 </c:choose>
 <div class="panel panel-primary">
     <div class="panel-heading">
+        <div class='display-icon pull-right'>
+            <span class="glyphicon ${'glyphicon-chevron-up'} pull-right"></span>
+        </div>
         <strong>${bodyTitle}</strong>
     </div>
-    <div class="panel-body">
+    <div class="panel-collapse collapse in">
```

Comment:
I see, perhaps you could try putting the <div class="panel-body"> element within the `<div class="panel-collapse collapse in">` element.